README.md

```diff
@@ -68,6 +68,7 @@ Supported Cars
 | Cadillac<sup>3</sup> | ATS 2018                   | Adaptive Cruise      | Yes     | Yes            | 0mph             | 7mph           | Custom<sup>7</sup>|
 | Chrysler             | Pacifica 2017-18           | Adaptive Cruise      | Yes     | Stock          | 0mph             | 9mph           | FCA               |
 | Chrysler             | Pacifica Hybrid 2017-18    | Adaptive Cruise      | Yes     | Stock          | 0mph             | 9mph           | FCA               |
+| Chrysler             | Pacifica 2019              | Adaptive Cruise      | Yes     | Stock          | 0mph             | 39mph          | FCA               |
```

Comment:
small thing: since it's alphabetic order, this should go one line up (before the hybrid)

Reply:
sure thing, done